UPGRADING.md

```diff
@@ -80,6 +80,8 @@ these [instructions](https://www.mongodb.com/docs/manual/core/capped-collections
 UTF-8 encoding. 
 <br>Note that this encoding is applied to all messages received by the input. A single input
 cannot handle multiple log sources with different encodings.
+- The permissions for what gets populated in the System dropdown menu were updated to more closely match the page that they link to. 
```

Comment:
Suggesting a small adjustment in wording for clarity, and also a link to the PR, in case a user wants to see which permissions/options have changed.
```suggestion
- The permissions for which options are populated in the System dropdown menu were updated to more closely match the page that they link to. See [graylog2-server#13188](https://github.com/Graylog2/graylog2-server/pull/13188) for details.
```